lightkurve/collections.py

```diff
@@ -85,6 +98,12 @@ def __repr__(self):
             result += '\n'
         return result
 
+    @property
+    def sector(self):
+        """(TESS-specific) the sectors of the lightcurves / target pixel files
+        """
+        return np.array([lc.sector for lc in self.data])
```

Comment:
I recommend implementing this as follows:
```suggestion
        return np.array([lc.meta.get('sector') for lc in self.data])
```
....so that we return an array of `None` if `lc.sector` does not exist.

Reply:
Ah, scrap that!  `self.data` may be a `TargetPixelFile` here, which does not have a `meta` dictionary yet, so it's fine the way you have it.